apps/voting/app/src/components/VoteDetail/DisputableActionStatus.js

```diff
@@ -22,7 +23,7 @@ function DisputableActionStatus({ vote, connectedAccount }) {
       <Box heading="Disputable Action Status">
         <Item>
           <Label>Status</Label>
-          <div>DisputableStatusTag</div>
+          <DisputableStatusLabel vote={vote} />
```

Comment:
With the changes to use `Map` below, you could now pass the status symbol to this component directly and guard if necessary:

```
const disputableStatus = vote.disputable && DISPUTABLE_VOTE_STATUS.get(vote.disputable.status)

......

{disputableStatus && <DisputableStatusLabel status={disputableStatus} />}
```

As we only care about the `status` we can avoid passing the entire vote which makes it a bit easier to reason about and keeps all of our concerns clear.